src/agent/rustjail/src/container.rs

```diff
@@ -669,7 +667,12 @@ fn do_init_child(cwfd: RawFd) -> Result<()> {
     // there's no uid entry in /etc/passwd, set "/" as the
     // home env.
     if env::var_os(HOME_ENV_KEY).is_none() {
-        let home_dir = utils::home_dir(guser.uid).unwrap_or_else(|_| String::from("/"));
+        let mut home_dir = String::from("/");
+        if let Ok(Some(user)) = User::from_uid(Uid::from_raw(guser.uid)) {
```

Comment:
Nit: I think you could rearrange this slightly to remove the need for `mut`.

Reply:
Hello @jodh-intel,

Thanks for your review, I was removed mut variable from code, PTAL :)